docs/reference/index-modules.asciidoc

```diff
@@ -76,14 +76,16 @@ breaking change].
 
     The +default+ value compresses stored data with LZ4
     compression, but this can be set to +best_compression+
-    which uses {wikipedia}/DEFLATE[DEFLATE] for a higher
-    compression ratio, at the expense of slower stored fields performance.
+    which uses {wikipedia}/Zstd[ZSTD] for a higher
+    compression ratio, at the expense of slower stored fields read performance.
     If you are updating the compression type, the new one will be applied
     after segments are merged. Segment merging can be forced using
     <<indices-forcemerge,force merge>>. Experiments with indexing log datasets
-    have shown that `best_compression` gives up to ~18% lower storage usage in
-    the most ideal scenario compared to `default` while only minimally affecting
-    indexing throughput (~2%).
+    have shown that `best_compression` gives up to ~28% lower storage usage and
+    better indexing throughput (up to ~10%) in the most ideal scenario compared
```

Comment:
I imagine that it can be faster at indexing than `default` in some cases via more batching, but also slower in other cases e.g. when index sorting is involved as some merging optimizations that copy data directly would no longer be applicable. I would rather rephase to suggest similar indexing rates, occasionally a bit slower or a bit faster depending on other options configured on the index?

Reply:
👍 - I will reword this. This is based on comparing elastic/logs track without logsdb with default codec (lz4) codec to  elastic/logs track without logsdb with best_compression (zstd)